### PR TITLE
flagmaster info.md PR

### PR DESCRIPTION
**Pull request by RayDeeUx**

### CODE REVIEW

**Comment by Su386yt** [comment]
@RayDeeUx did you notice who asked you

File: info.md [hunk 1]
```diff
@@ -23,60 +21,23 @@ If you don't know how to update the list, consider watching this [video](https:/
...
-    "discord_invite_code": "v4PU3WeH7z"
+    "latest_version_release_date" : "01 January 1970"
 },
 ```
 
-This object has a ``prank_sound`` boolean,
+This object also has a ``prank_sound`` boolean,
 it's a kill switch for the sound prank feature, incase something goes wrong.
 
-Format:
```
Comment (Su386yt) [inline info.md:45]: Tf did you do lmao

File: info.md [hunk 2]
```diff
@@ -105,7 +65,7 @@ Example:
 }
 ```
 
-### [/data/constants/mathematical_hoes.json](/data/constants/mathematical_hoes.json)
```
Comment (Su386yt) [inline info.md:108]: Why are things moved around

**Review by Su386yt — CHANGES_REQUESTED** [review]
Someone forgot to rebase

**Comment by Su386yt** [comment]
> flagmaster info.md PR

You do realize flagmaster is not the one who asked you to do this

**Comment by RayDeeUx** [comment]
closed, see #81

**Pull request closed without merging**